addons/mrp/static/tests/mrp_document_kanban.test.js

```diff
@@ -0,0 +1,169 @@
+import { describe, test } from "@odoo/hoot";
+import { assertSteps, click, contains, openView, registerArchs, start, startServer, step } from "@mail/../tests/mail_test_helpers";
```

Comment:
I think this line should have been wrapped by prettier, 132 chars is too long according to our guidelines.